### ISSUE TITLE
runCommand() runs function(chunks, mode), not function(mode, chunks)

### ISSUE THREAD

**Issue by jywarren** [bevacqua/woofmark#29]
Unless I've made some mistake:

``` js
editor.runCommand(function(mode, chunks) { console.log(mode, chunks) })
> HtmlChunks {before: "<p>", startTag: "", selection: "", endTag: "", after: "lalalala</p>↵"…} "wysiwyg"
```

**Comment by bevacqua** [comment]
You are right. We should probably leave it be and fix the documentation.

**Comment by jywarren** [comment]
Fixed in #31.
